File: C++/I/05-FORMS/2/form3.cpp
#include "form3.h"
#include "ui_form3.h"

Form3::Form3(QWidget *parent) :
    QDialog(parent),
    ui(new Ui::Form3)
{
    ui->setupUi(this);
    setFixedSize(size());
}

Form3::~Form3()
{
    delete ui;
}

File: Python/I/05-FORMS/3/form3.py
from ui_form3 import *
from PyQt5.QtCore import *
from PyQt5.QtGui import *
from PyQt5.QtWidgets import *

class Form3(Ui_Form3, QDialog):    
    def __init__(self, parent):
        super(self.__class__, self).__init__(parent)
        self.setupUi(self)
        self.setFixedSize(self.size())
